components/dada-ir/src/code/validated.rs

```diff
@@ -258,6 +258,9 @@ pub enum ExprData {
     /// `expr.give`
     Give(Place),
 
+    /// `(expr)`
+    Parenthesized(Expr),
```

Comment:
Hmm, I think we shouldn't have this in `Validated`....

Reply:
...it belongs in `Syntax`, but `Validated` is supposed to be kind of only the things that have meaning to the compiler